tests/general/AnnotationBearTest.py

```diff
@@ -112,3 +114,14 @@ def test_combined_strings(self):
             self.assertIn(string2, results[0].contents['strings'])
             self.assertIn(string3, results[0].contents['strings'])
             self.assertEqual(results[0].contents['comments'], ())
+
+    def test_external_coalang(self):
+        section3 = Section("")
```

Comment:
could we not give variables numbers? Don't you already create a section in setUp that you can just fll here? It's mutable.